automatic/octave.portable/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,41 @@
+﻿$version = '4.2.2'
+
+$toolsDir       = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$progDir        = "$toolsDir\octave"
+
+$osBitness      = Get-ProcessorBits
+
+$packageArgs = @{
+    PackageName     = 'octave.portable'
+    UnzipLocation   = $toolsDir
+    Url             = 'https://ftp.gnu.org/gnu/octave/windows/octave-4.2.2-w32.zip'
+    Url64           = 'https://ftp.gnu.org/gnu/octave/windows/octave-4.2.2-w64.zip'
+    Checksum        = '1446997f71ed8fee725d17e0850087f14b59fba36aacbc0fea799943a6eaad20'
+    Checksum64      = 'f41d561c53fd193510f337dd379c0e0853df830fc5dced4dd98a6c4d65535117'
+    ChecksumType    = 'sha256'
+    ChecksumType64  = 'sha256'
+}
+Install-ChocolateyZipPackage @packageArgs
```

Comment:
I see you have done a mixed work for embedded package and for a remote install package.

Please update the whole script to truly embed the package.

Reply:
The software distribution packages are too large to embed; download is preferred.